recipes/pyutilib/meta.yaml

```diff
@@ -97,7 +97,3 @@ about:
   license: BSD License
   summary: 'PyUtilib: A collection of Python utilities'
 
-extras:
-  recipe-maintainers:
-    - whart222
```

Comment:
I omitted the 'extras' block because it was causing a test failure.  Is
that expected?

--Bill

On Tue, Apr 12, 2016 at 3:14 PM, Filipe <notifications@github.com> wrote:

> In recipes/pyutilib/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/331#discussion_r59436118>
> :
>
> > @@ -97,7 +97,3 @@ about:
> >    license: BSD License
> >    summary: 'PyUtilib: A collection of Python utilities'
> >
> > -extras:
> > -  recipe-maintainers:
> > -    - whart222
>
> You have to add this back because the feedstock will be generated using
> this info.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/331/files/3cdbb10e87b8f33fb611b768c1e8ece26d9911f1..79e192f0ac0d17b9554f605598a57a1d206c4aaf#r59436118>
>